core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
i'm not arguing about the bindings, it is best to have just one as i mentioned above.
i'm just saying that we also need to match empty BindingProps.